README.md

```diff
@@ -496,6 +496,58 @@ onNativeEvent(({event}) => {
 -   `tappedNavigationBarBackButton`: fired when the user taps on the back button
     of the native Navigation Bar. Allowed response actions: `default`
 
+### checkPermissionStatus
+
+Returns true if the app has the specific notifications permissions. You have to
+pass feature and required params for this request.
+
+-   Available for app versions 11.4 and higher
+
+You have these features avalaibles:
+
+-   `notifications`
+
+```typescript
+checkPermissionStatus: (feature: string, params?: {[key: string]: string},) => Promise<boolean>;
+```
+
+#### Example
+
+```javascript
+import {checkPermissionStatus} from '@tef-novum/webview-bridge';
+
+checkPermissionStatus('notifications',{channelId: 'default'}).then((hasPermissions) => {
+    console.log(hasPermissions);
+}).catch(err => {
+    console.error(err);
+};
+```
+
+### internalNavigation
+
+Init an internal and native navigation to a device specific feature
+
+-   Available for app versions 11.4 and higher
+
+You have these features avalaibles:
```

Comment:
`You have these features available:` (singular)

or just:

`Available features:`

Same in the other method